web/index.html

```diff
@@ -46,8 +40,7 @@ <h3 class="sub-header">
                 </h3>
             </div>
 
-            <a class="github-link" style="position: absolute; top: 40px; left: 40px"
-                href="https://github.com/expitau/InfiniteCraftWiki" target="_blank">
+            <a class="github-link" href="https://github.com/expitau/InfiniteCraftWiki" target="_blank">
```

Comment:
moved some styling elements into the github-link class that should have already been there